web/src/search_suggestion.ts

```diff
@@ -90,7 +90,57 @@ function check_validity(
 }
 
 function get_suggestion_parts(operators: NarrowTerm[]): Suggestion[] {
-    return [{description_html: Filter.search_description_as_html(operators)}];
+    const valid_operators = new Set(["dm", "sender", "dm-including", "pm-with"]);
```

Comment:
Is `valid` the right way to describe these? I would think something like `const person_operators` might be clearer?